### PR TITLE
Feat/welcome mail

### PR DESCRIPTION
**Pull request by Juliocho990**
## Summary

* Nodemailer was user to sent email within the `createUser` controller.
* Tests were added.

## Known Issues

* None

## Trello Card

https://trello.com/b/QR9JDanI/training-express-js-juli%C3%A1n-lopera

### CODE REVIEW

File: app/controllers/users.js [hunk 1]
```diff
@@ -3,18 +3,22 @@ const errors = require('../errors');
...
     const hashCode = await createHash(userData.password);
     if (hashCode) userData.password = hashCode;
     await UserServices.createUser(userData);
+    await MailerService.sendWelcomeEmail(userData);
     return res.status(201).send({
       data: { name: userData.name },
       message: CREATED
     });
   } catch (err) {
+    logger.info('createUser: ', err);
```
Comment (blueJayWays) [inline app/controllers/users.js]: You're logging the error or the created user?

File: app/services/mailer.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+const mailer = require('nodemailer');
+const { EMAIL_SENDER, EMAIL_GENERAL_ERROR } = require('../../config/constants');
+const logger = require('../logger');
+
+const welcomeMessage = (name, lastName, email) => ({
+  from: `<${EMAIL_SENDER}>`,
+  to: email,
+  subject: 'Welcome ✔',
+  text: `Welcome ${name} ${lastName}`,
+  html: `<h1>Witter API</h1>
```
Comment (blueJayWays) [inline app/services/mailer.js]: If your HTML is too large, is necessary to create another file, then import it.

**Review by blueJayWays — CHANGES_REQUESTED** [review]

File: app/services/mailer.js (hunk 2 above)
Comment (Juliocho990) [reply app/services/mailer.js]: Great approach.

File: app/controllers/users.js (hunk 1 above)
Comment (Juliocho990) [reply app/controllers/users.js]: Fixed

**Comment by blueJayWays** [comment]
Good!

**Pull request merged**